tools/azure-pipelines/jobs-template.yml

```diff
@@ -129,6 +129,21 @@ jobs:
     condition: not(eq('${{parameters.test_pool_definition.name}}', 'Default'))
     displayName: Cache Maven local repo
 
+  - task: Cache@2
+    inputs:
+      key: '"$(module)" | $(TESTCONTAINER_CACHE_KEY)'
+      path: $(TESTCONTAINER_CACHE_FOLDER)
+      cacheHitVar: TESTCONTAINER_CACHE_HIT
+    continueOnError: true
+    condition: not(eq('${{parameters.test_pool_definition.name}}', 'Default'))
+    displayName: Cache Testcontainer docker images
```

Comment:
nit: Its not just Testcontainer images, so the name of this task and other variables is ever-so-slightly misleading

Reply:
I'm not sure what would be more discriptive :sweat_smile: Perhaps "Testing Containers"?

Reply:
Why not "Docker images"?